.github/workflows/reusable_swap_functional_tests.yml

```diff
@@ -39,6 +39,10 @@ on:
         required: false
         default: "develop"
         type: string
+      branch_for_solana:
+        required: false
+        default: "wip/fbeutin/swap_rebase_post_refacto"
```

Comment:
Keeping the wip branch ?

Reply:
Yes because the solana swap is not yet on develop